sample-apps/react-native/dogfood/src/components/ActiveCall.tsx

```diff
@@ -12,9 +12,13 @@ import { useOrientation } from '../hooks/useOrientation';
 import { Z_INDEX } from '../constants';
 import { TopControls } from './CallControlls/TopControls';
 import { useLayout } from '../contexts/LayoutContext';
-import { useToggleCallRecording } from '@stream-io/video-react-bindings';
+import {
+  useCallStateHooks,
+  useToggleCallRecording,
+} from '@stream-io/video-react-bindings';
```

Comment:
```suggestion
} from '@stream-io/video-react-native-sdk';
```

We shouldn't be importing from the bindings package here. This package is meant to be used only internally within the SDK. I expected ESLint to complain about this.